src/api/routes/debug.ts

```diff
@@ -374,5 +374,40 @@ export function createDebugRouter(db: DataStore): RouterWithAsync {
     res.set('Content-Type', 'text/html').send(txWatchHtml);
   });
 
+  router.postAsync('/faucet', (req, res) => {
+    try {
+      const { address } = req.query;
+      if (!process.env.FAUCET_PRIVATE_KEY) {
+        res.json({
+          success: false,
+          error: 'Faucet not setup',
+        });
+        return;
+      }
+
+      const tx = makeSTXTokenTransfer(
+        address,
+        new BN(10e3),
+        new BN(10),
+        process.env.FAUCET_PRIVATE_KEY,
+        {
+          nonce: new BN(0),
```

Comment:
What is the intended usage here? Shouldn't this use the core-node RPC to retrieve the nonce for `FAUCET_PRIVATE_KEY` and then to submit the transaction?

Reply:
Ah yeah my bad - forgot to add that. This faucet was written before the RPC API was available. Will add.

Reply:
Good catch - added it back